internal/core/queries/decode_vin.go

```diff
@@ -89,8 +89,13 @@ func (dc DecodeVINQueryHandler) Handle(ctx context.Context, query mediator.Messa
 	)
 
 	metrics.Success.With(prometheus.Labels{"method": VinRequests}).Inc()
+	tx, err := dc.dbs().Writer.BeginTx(ctx, nil)
```

Comment:
You might want serializable, I don't know. Is read committed enough for you?

Reply:
yeah, left that while i did some research. 
in agreement, ended up going for serializable, thanks for validating